torch_geometric/utils/_scatter.py

```diff
@@ -243,11 +243,20 @@ def group_argsort(
             (default: :obj:`False`)
         stable (bool, optional): Controls the relative order of equivalent
             elements. (default: :obj:`False`)
+
+    Example:
+        >>> src = torch.tensor([0, 0, 1, 1, 1, 1, 2, 2, 2])
+        >>> index = torch.tensor([0, 1, 5, 4, 3, 2, 6, 7, 8])
```

Comment:
I assume you need to swap `src` and `index` here.

Reply:
ah yes! PTAL at 563c007 :)